ERCS/erc-TBA.md

```diff
@@ -0,0 +1,819 @@
+---
```

Comment:
```suggestion
---
eip: 7794
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.